conda_env/specs/binstar.py

```diff
@@ -1,16 +1,18 @@
 # Copyright (C) 2012 Anaconda, Inc
 # SPDX-License-Identifier: BSD-3-Clause
-import importlib
+from __future__ import annotations
+
+from functools import cached_property
```

Comment:
Unfortunately functool.cached_property should never be used: https://github.com/python/cpython/issues/87634 . Hopefully it won't matter here.

Reply:
My guess is that it won't matter given the low impact of the code that is decorated. But we might be able to replace it with `@lru_cache` instead like we do elsewhere in the code. Although that has [its own problems](https://rednafi.github.io/reflections/dont-wrap-instance-methods-with-functoolslru_cache-decorator-in-python.html).